kratos/solving_strategies/strategies/explicit_solving_strategy_runge_kutta.h

```diff
@@ -188,6 +422,16 @@ class ExplicitSolvingStrategyRungeKutta4 : public ExplicitSolvingStrategy<TSpars
     ///@name Operations
     ///@{
 
+    void Initialize() override
+    {
+        BaseType::Initialize();
+        BaseType::GetModelPart().GetProcessInfo().SetValue(TIME_INTEGRATION_THETA, 0.0);
+    }
+
+    static constexpr unsigned int Order()
+    {
+        return TButcherTableau::Order();
+    }
```

Comment:
In here you're extending the public API of the strategy. Is there any technical reason behind? If it is only to inquiry the strategy order I would revert it.

Reply:
No, no reason for it. Deleting it.